sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala

```diff
@@ -154,4 +155,32 @@ class HiveExternalCatalogSuite extends ExternalCatalogSuite {
     catalog.createTable(hiveTable, ignoreIfExists = false)
     assert(catalog.getTable("db1", "spark_29498").owner === owner)
   }
+
+  test("SPARK-30868 throw QueryExectionException if run hive query failed") {
```

Comment:
`run hive query failed` is misleading, how about
```
SPARK-30868: throw an exception if HiveClient#runSqlHive fails
```

Reply:
good